guides/micronaut-oauth2-github/groovy/src/main/groovy/example/micronaut/GithubRepo.groovy

```diff
@@ -0,0 +1,17 @@
+package example.micronaut
+
+import groovy.transform.CompileStatic
+import io.micronaut.core.annotation.Creator
+import io.micronaut.core.annotation.Introspected
+
+@CompileStatic
+@Introspected
+class GithubRepo {
+
+    final String name
+
+    @Creator
```

Comment:
Is `@Creator` necessary? If you want to keep, we need to add a callout and explain it to the user.